packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.controller.js

```diff
@@ -1,13 +1,18 @@
-import { NSX_RESOURCES } from './dedicatedCloud-datacenter-manage-nsx.constants';
+import {
+  NSX_RESOURCES,
+  TRACKING_SUFFIX,
+} from './dedicatedCloud-datacenter-manage-nsx.constants';
 import { EDGES_SIZES } from '../../../datacenters/datacenter.constants';
 
 export default class {
   /* @ngInject */
-  constructor(DedicatedCloud, $translate, coreURLBuilder) {
+  constructor(DedicatedCloud, $translate, coreURLBuilder, TRACKING_PREFIX) {
```

Comment:
use routing / component binding to load the property instead of injecting it ine the constructor
```suggestion
  constructor(DedicatedCloud, $translate, coreURLBuilder) {
```